views/018_component_triggers.sql

```diff
@@ -0,0 +1,22 @@
+CREATE
+OR REPLACE FUNCTION handle_component_updates () RETURNS TRIGGER AS $$
+BEGIN
+  IF OLD.deleted_at IS NULL AND NEW.deleted_at IS NOT NULL THEN
+    DELETE FROM team_components WHERE component_id = OLD.id;
```

Comment:
I am not sure we should be doing this - If I am looking at a deleted component, I still want to see what it was related to before it was deleted. 

A relationship should only be deleted when it is no longer true e.g. An EBS volume is detached from an ec2instance - in which case the component/config relationship.deleted_at should reflect the detach time

Reply:
@adityathebe  / @yashmehrotra  ^